### PR TITLE
Adding TPM support to U-boot for RPi4

### PR DESCRIPTION
**Pull request by Insei**
Add TPM slb9670 module support to U-boot for RPi4 via overlay.

### CODE REVIEW

File: pkg/u-boot/rpi/config.txt [hunk 1]
```diff
@@ -25,7 +25,7 @@ avoid_warnings=1
 
 # This overlay fixes divergences between the RPi device tree and what
 # upstream provides
```
Comment (eriknordmark) [inline pkg/u-boot/rpi/config.txt:27]: Is this comment still correct given that you changed the overlay?

Comment (Insei) [reply pkg/u-boot/rpi/config.txt:27]: @eriknordmark No, it's not correct, I missed it, I'll change the comment.

**Comment by eriknordmark** [comment]
@mikem-zed can you review this?

**Comment by rene** [comment]
@Insei , I'd like to recommend to break this PR into two, would that be possible?

1. PR only to update U-Boot to 2022.10
2. PR to add TPM support to U-Boot 

It breaks even more logically the changes and makes more easy for those that just want to test new U-Boot, without care about TPM (I'm interested in doing so)....

**Comment by Insei** [comment]
@rene Yes, it will be so, that is why the PR hangs in the WIP status.

**Comment by Insei** [comment]
First we need to merge the PR: https://github.com/lf-edge/eve/pull/2933
Because in that PR we are building dtb from u-boot sources, and u-boot adds a __symbols__ node to dtb, which is used by the overlay added here.

File: pkg/u-boot/rpi/config.txt [hunk 2]
```diff
@@ -24,6 +24,13 @@ enable_uart=1
 # this overlay overwrites the SMBIOS information.
 #dtoverlay=raspberrypi-uno-220
 
+# This overlay adds support for the slb9670 tpm module to uboot
+# Some boards can have defferent ce(address), we can set ce by param,
```
Comment (eriknordmark) [inline pkg/u-boot/rpi/config.txt:28]: Please fix yetus/codespell issue.

**Review by eriknordmark — COMMENTED** [review]
Is this ready for review, or still WIP?

**Comment by Insei** [comment]
All tested and works

**Review by eriknordmark — APPROVED** [review]
LGTM

**Pull request merged**